infra/cmake/modules/ExternalSourceTools.cmake

```diff
@@ -103,7 +103,13 @@ function(ExternalSource_Download PREFIX)
 
     message(STATUS "Extract ${PREFIX}")
     execute_process(COMMAND ${CMAKE_COMMAND} -E tar xfz "${DOWNLOAD_PATH}"
+                    RESULT_VARIABLE EXITCODE
                     WORKING_DIRECTORY "${TMP_DIR}")
+
+    if(NOT EXITCODE EQUAL 0)
+      message(FATAL_ERROR "Extract ${PREFIX} - fail")
+    endif(NOT EXITCODE EQUAL 0)
+
```

Comment:
This pattern already used in https://github.com/Samsung/ONE/blob/31e5b3a6ade428688a3e96b891500a1cc0c841b1/compiler/tf2circle-model-test/CMakeLists.txt#L32-L39

It may better introduce general wrapper for `execute_process`? :smile: